lib/money/splitter.rb

```diff
@@ -77,7 +77,6 @@ def [](index)
           return money
         end
       end
-      nil
```

Comment:
![image](https://github.com/user-attachments/assets/926a4792-abc2-4354-864f-a9f2aa11b4c6)

Reply:
<img width="766" alt="Screenshot 2025-04-11 at 9 04 56 AM" src="https://github.com/user-attachments/assets/9d5cd984-8033-46dd-895c-dd64c268dbbc" />

CI runs simplecov as and checks that we have 100% coverage, strange that it didn't pick up on it 🤔 Are you sure you ran all the tests?

Reply:
yes I think we're missing a test for the case where this returns nil. Strange that this didn't make CI fail before ruby 3.4

Reply:
`the last expression would be split.each do ... end. The each method on a Hash returns the hash itself`